sample_fast_api/apps/auth/authentication.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime, timedelta
+from typing import Optional
+
+from fastapi import Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+from jose import JWTError, jwt
+
+from sample_fast_api.apps.users.models import User
+from sample_fast_api.apps.users.schemas import pwd_context
+from sample_fast_api.config import settings
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
+
+
+def validate_token(token: str = Depends(oauth2_scheme)):
+    credentials_exception = HTTPException(
```

Comment:
Maybe this variable can stay outside of this method, to be created once at app loading

Reply:
:+1: